### PR TITLE
FIREFLY-938: ZTF proprietary metadata is not returned in GUI

### PR DESCRIPTION
**Pull request by loitly**
Ticket: https://jira.ipac.caltech.edu/browse/FIREFLY-938
Test: https://firefly-938-ztf-proprietary.irsakudev.ipac.caltech.edu/applications/ztf/

Should be fixed now.  Follow ticket description to test.

### CODE REVIEW

**Comment by lrebull** [comment]
I confirm, this works; I can see data from 2/20/2022. I had some trouble (error messages) when logging in, but I did finally successfully log in, and I can see those data.

**Comment by lrebull** [comment]
(and before logging in, i could not see those data)

**Review by wmiipac — APPROVED** [review]
Code make sense to me for each ibe query url pass the authentication credential. 
Tested searches and download with the proprietary data, works well.
Thanks!

**Review by ejoliet — APPROVED** [review]
Tested. Looks good! 

(Was the Deprecated ibe file uploader class meant for IBE multi position search or was for something else)

**Comment by loitly** [comment]
> Tested. Looks good!
> 
> (Was the Deprecated ibe file uploader class meant for IBE multi position search or was for something else)

It was just an unnecessary layer of abstraction design element that's not used.  I would've removed it.  But, since this is a hot patch fix, I don't want to put too much changes into it.

File: src/firefly/java/edu/caltech/ipac/astro/ibe/IbeFileUploader.java [hunk 1]
```diff
@@ -12,6 +12,7 @@
  *
  * @author loi
  * @version $Id: $
+ * @deprecated
```
Comment (robyww) [inline src/firefly/java/edu/caltech/ipac/astro/ibe/IbeFileUploader.java:15]: If this file is not used why not just removed it.

File: src/firefly/java/edu/caltech/ipac/firefly/server/query/ibe/IbeFileUploaderImpl.java [hunk 2]
```diff
@@ -27,6 +27,7 @@
  * Implementing IbeFileUploader using HttpClient via MultiPartPostBuilder
  * @author loi
  * @version $Id: $
+ * @deprecated
```
Comment (robyww) [inline src/firefly/java/edu/caltech/ipac/firefly/server/query/ibe/IbeFileUploaderImpl.java:30]: If this file is not used why not just removed it.

**Review by robyww — APPROVED** [review]
Looks good. I don't see any reason not to remove the deprecated files.

**Pull request merged**